src/lxc/lxccontainer.h

```diff
@@ -1017,6 +1017,22 @@ int list_active_containers(const char *lxcpath, char ***names, struct lxc_contai
  */
 int list_all_containers(const char *lxcpath, char ***names, struct lxc_container ***cret);
 
+struct lxc_log_conf {
+	const char *name;
+	char *file;
+	char *priority;
+	const char *prefix;
+	int quiet;
+	const char *lxcpath;
+};
+
+/*!
+ *\brief Initialize the log
+ *
+ *\param log lxc log configuration.
+ */
+int lxc_log_init(struct lxc_log_conf log);
```

Comment:
It's just good practice to not copy the whole struct into the function. By using a pointer the function becomes a lot more lightweight. If you really think that we'll never change anything that is inside the struct then make it `const struct *log`. But the thing about a struct is that you can extend it later on.